### ISSUE TITLE
New chapter proposal - Synthetic difference in differences

### ISSUE THREAD

**Issue by dbalabka** [matheusfacure/python-causality-handbook#227]
Thanks for a great book! It gives every intuitive way to understand the Causal Inference topic. All illustrations are terrific and make studying fun :)

After having read your book, I've found the interesting approach that combines Synthetic Controls and Difference-in-Difference [1]. I think it might be a logical addition to your book as a new chapter. Also, there are Python [2] and R [3] implementations that might help to provide code examples.

References
[1] [Arkhangelsky, D., Athey, S., Hirshberg, D.A., Imbens, G.W. and Wager, S., 2019. Synthetic difference in differences (No. w25532). National Bureau of Economic Research.](https://www.nber.org/system/files/working_papers/w25532/w25532.pdf)
[2] Causal Inference Using Synthetic Difference in Differences with Python | by Masa Asami | Python in Plain English. (n.d.). Retrieved May 10, 2022, from https://python.plainenglish.io/causal-inference-using-synthetic-difference-in-differences-with-python-5758e5a76909
[3] Synthetic Difference-in-Difference Estimation • synthdid. (n.d.). Retrieved May 10, 2022, from https://synth-inference.github.io/synthdid/

**Comment by matheusfacure** [comment]
Thanks for the references! I have a plan to expand the chapters on Panel Data to include not only that, but also some other models from this great course here: https://www.youtube.com/playlist?list=PLo0lw6BstMGZQqx_r1GnOETkFYihCgve9

**Comment by dbalabka** [comment]
@matheusfacure It is great news and looking forward to reading it :)

**Comment by dbalabka** [comment]
@matheusfacure, I'm, currently, working on SEO experiments. The general idea is described in the Airbnb's article:
https://medium.com/airbnb-engineering/experimentation-measurement-for-search-engine-optimization-b64136629760
Probably, it might be a good example of DiD application in the industry that can be mentioned in the article.

**Comment by matheusfacure** [comment]
Here is the branch. It only has code (no text yet), but you can get the idea on how to implement SDID
